study-builder/studies/study-example.conf

```diff
@@ -29,6 +29,33 @@
     "shareParticipantLocation": "bool",
   },
 
+  # Configuration of the statistics data available for the study
+  "statistics": [
+    {
+      # Number of participants enrolled. Divided by types (self, governed)
+      "type": "PARTICIPANTS"
+    },
+    {
+      # Shows answers distribution for the specific picklist question. Contains also picklist option details with i18n applied
+      "type": "DISTRIBUTION",
+      "stableId": "REGISTRATION_COUNTRY"
+    },
+    {
+      # Number of subscriptions
+      "type": "MAILING_LIST"
+    },
+    {
+      # Shows how many specific answers were given for the specific question. Supported question types: picklist, boolean
+      "type": "SPECIFIC_ANSWER",
+      "stableId": "PERFORM_DNA",
+      "value": "true"
```

Comment:
Having an example is nice, but might be more desirable to have documentation. E.g.:
```
"stableId": "string",
"value": "bool"
```

Reply:
Changed